CMakeLists.txt

```
@@ -56,6 +56,10 @@ option(ZAP_TEST
        "ZAP fuzz test using https://www.zaproxy.org/docs/docker/api-scan/" OFF
 )
 option(BUILD_SMALLBANK "Build SmallBank sample app and clients" ON)
+option(ENABLE_JS_GOV "Disable JS Governance" ON)
```

```suggestion
option(ENABLE_JS_GOV "Enable JS Governance" ON)
```

Should this be off by default?

---

Started out that way, but now it's on by default and only disabled in release mode.